main/unbound/unbound.initd

```diff
@@ -22,36 +23,22 @@ depend() {
 	after auth-dns
 }
 
-checkconfig() {
-	checkpath --directory /var/run/unbound
-	UNBOUND_PIDFILE=$("${UNBOUND_CHECKCONF}" -o pidfile "${UNBOUND_CONFFILE}")
-	return $?
-}
-
 configtest() {
-	ebegin "Checking ${SVCNAME} configuration"
-	checkconfig
-	eend $?
-}
-
-start() {
-	checkconfig || return $?
-	ebegin "Starting ${SVCNAME}"
-	start-stop-daemon --start --pidfile "${UNBOUND_PIDFILE}" \
-		--exec "${UNBOUND_BINARY}" -- -c "${UNBOUND_CONFFILE}"
-	eend $?
+	einfo "Checking: $cfgfile" # don't show errors twice
+	pidfile=$($checkconf -o pidfile "$cfgfile" 2>/dev/null)
```

Comment:
Why do you do that? Unbound spreads its pidfiles over FS even when you start it without `-d`? :(

Reply:
to prevent:
```
root@dns2 [/etc/init.d]# rc-service unbound start
 * Checking: /etc/unbound/unbound.conf
Usage: checkpath [options] path1 [path2] [...]

Options: [ dDfFpm:o:WChqVv ]
  -d, --directory                   Create a directory if not exists
  -D, --directory-truncate          Create/empty directory
  -f, --file                        Create a file if not exists
  -F, --file-truncate               Truncate file
  -p, --pipe                        Create a named pipe (FIFO) if not exists
  -m, --mode <arg>                  Mode to check
  -o, --owner <arg>                 Owner to check (user:group)
  -W, --writable                    Check whether the path is writable or not
  -h, --help                        Display this help output
  -C, --nocolor                     Disable color output
  -V, --version                     Display software version
  -v, --verbose                     Run verbosely
  -q, --quiet                       Run quietly (repeat to suppress errors)
 * Using pidfile: 
 * Starting unbound daemon ... * command_background option used but no pidfile specified
                                                                                               [ !! ]
 * ERROR: unbound failed to start
```

Reply:
from:
```
root@dns2 [/etc/init.d]# grep -A5 ^configtest unbound
configtest() {
	einfo "Checking: $cfgfile" # don't show errors twice
#	pidfile=$($checkconf -o pidfile "$cfgfile" 2>/dev/null)
	$checkconf -f $cfgfile 1>/dev/null
	return $?
}
```

Reply:
this `PR` is correct & works:
```
root@dns2 [~]# rc-service unbound start
 * Caching service dependencies ... [ ok ]
 * Checking: /etc/utnbound/unbound.conf
 * Using pidfile: /run/unbound.pid
 * Starting unbound daemon ... [ ok ]
```